apps/prairielearn/python/README.md

```diff
@@ -0,0 +1,5 @@
+# Python Directory
+
+This directory contains all the shared python code. Elements can import from this directory to use shared code. You should not import from `test` or from `prairielearn.internal`, as they are not guaranteed to have a stable API.
+
+All element code is run in the context of zygote.py, so it will be run from this directory.
```

Comment:
```suggestion
All element code is run in the context of `zygote.py`, so it will be run from this directory.
```

The fact that questions are executed by the Zygote isn't directly connected to why things can e.g. import `prairielearn`. Questions are executed with their working directory as the question. The zygote just adds its own working directory to Python's path:

https://github.com/PrairieLearn/PrairieLearn/blob/b0d86c51d43b02de170b35200768473bd00cef31/apps/prairielearn/python/zygote.py#L272-L275

Reply:
Changed the text to reflect this distinction in 2a955b47aaba80ca142730b9a9cdb229eddf92a8.